README.md

```diff
@@ -93,3 +93,109 @@ allow_project_configs=true
 org=my_okta_org
 user=my_user
 ```
+
+## How to use bmx and the AWS cli 🚲 - Opinionated Workflow
+
+### Pre-requisites
+- [Install the aws cli](https://aws.amazon.com/cli/)
+- Install and configure the d2l bmx tool
+  - [bmx repo](https://github.com/Brightspace/bmx)
+  - [Confluence guide on how to install bmx](https://desire2learn.atlassian.net/wiki/spaces/TH/pages/1011745036/Installing+and+using+BMX)
+
+
+### Step 1 - Add this alias to your bashrc
+
+```bash
+# bashrc
+export OKTA_USER={YOUR_OKTA-USER_HERE}
```

Comment:
Not sure if this is the norm, how I have it setup is a file in ~/.bmx/config with:
```
org = d2l
user = awijayathilaka
allow_project_configs = true
```
to avoid having to specify org and user everywhere

Reply:
Yeah I understand that the config file is meant to store the flag values, but I remember when going through the setup, it made it more difficult to understand what it was actually doing and how that would interact with the aws-cli and env variables. Perhaps we would need to just add more detail on the config file works? For example, if I want two different accounts setup, do I do that in the one config file?

Reply:
Ah, I guess this would just be for the flags that would be common to all accounts, i.e. org and user

Reply:
Yeah, it reads both the file at ~/.bmx/config, and any .bmx (composing them together, with local file taking precedence) files in working directory (or up the chain from there) not the most intuitive behaviour.

Reply:
Updated the setup to suggest using the profile for the two flags
https://github.com/Brightspace/bmx/pull/212/commits/aa54f4207b26faabcdb33d4d62d7678b0a9e3141